### DISCUSSION TITLE
The digital Accessibility of GitHub Product

### DISCUSSION THREAD

**Discussion by StefanXhunga** [community/community#54465] [Accessibility] [answered]
### Select Topic Area

Question

### Body

I know the truth is:
GitHub is an extensive code repository that has become popular with developers and companies hosting their projects, documentation, and code.  Apple, Amazon, Google and many other big tech companies use GitHub.  Microsoft is the main contributor to the site and has more than 1,000 employees actively pushing code to the GitHub repository.  Microsoft even hosts its original Windows File Manager source code on GitHub.  
  Microsoft has been rapidly investing in open source technology since Satya Nadella took over as CEO.  Microsoft has open sourced PowerShell, Visual Studio Code, and the Microsoft Edge JavaScript engine.  Microsoft also partnered with Canonical to bring Ubuntu to Windows 10 and acquired Xamarin to help develop mobile apps.

**Comment by samus-aran** [comment, marked as answer]
Hi @StefanXhunga ! We hope you found what you were looking for. ✨We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
